requirements.txt

```diff
@@ -48,7 +48,7 @@ pytest-metadata==1.7.0
 python-dateutil==2.7.5
 python-jose==2.0.2
 pytz==2018.7
-PyYAML==3.13
+PyYAML>=4.2b2
```

Comment:
all other requirements seem to be locked to a version. Okay that this one isn't?

Reply:
I think we're good. The requirements.txt was generated with `pip freeze`. We probably need to go through and figure out what we need and what we don't. I think there's some old stuff in here from when we were still trying to decide directions.